### DISCUSSION TITLE
Export Checkpoint Model

### DISCUSSION THREAD

**Discussion by timondurot** [coqui-ai/STT#2325] [Q&A] [answered]
I need to export the model I just trained somehow. I just have a bunch of files in my checkpoint directory. Any way to convert to .tflite?

**Comment by HarikalarKutusu** [comment, marked as answer]
You can use the export command. See:

https://stt.readthedocs.io/en/latest/EXPORTING_MODELS.html

or

https://stt.readthedocs.io/en/latest/playbook/DEPLOYMENT.html#exporting-a-tflite-model

**Reply by wasertech** [reply]
Since v1.4.0 (only inside the docker image), you need to use the virtual environment `/tflite-venv` to export your models.

See #2221 and #2220 for more information

**Comment by moonpetaldev** [comment]
what tf...
